src/Utilities/FlowAnalysis/Analysis/TaintedDataAnalysis/SinkKind.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Analyzer.Utilities.FlowAnalysis.Analysis.TaintedDataAnalysis
+{
+    internal enum SinkKind
+    {
+        Sql,
```

Comment:
>Sql, [](start = 8, length = 4)

Are you planning to add more fields here?

Reply:
Yes.  And er, actually use it in a meaningful manner.